### PR TITLE
feat(postman): prepared postman collection to R24.05 standard

### PR DESCRIPTION
**Pull request by tom-rm-meyer-ISST**
## Description

This PR updates the postman collection for integration tests (local/postman):
- Changed EDC catalog requests, to one request with checks for DTR, PartTypeInformation and ItemStock Submodel assets
- Added DTR incl ShellDescriptor evaluation for PartTypeInformation and ItemStock
- Added demand for customer setup
- Added production for supplier setup
- Restructured MAD folder per partner setup into MAD, Stock, Production, Demand
- Updated README.md with test data

Sidenote: While creating the collection, I run already into the following potential issues (please check):
- Posting an existing demand, creates a new one (do we maybe miss a business-wise check?)
- Posting a production with customer order id, customer order position id and no supplier id fails
- Getting productions lead to an error
- Posting a demand, needs the supplier location mandatory - at least the [samm](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.short_term_material_demand/1.0.0/gen/ShortTermMaterialDemand.html) would have it optional.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by tom-rm-meyer-ISST** [comment]
Restructured a little to already 
- check contract offers for demand and production
- check submodelDescriptors for demand and production

File: local/postman/puris-integration-test.postman_collection.json
[the remark this answers is not in the document]
Comment (tom-rm-meyer-ISST) [reply local/postman/puris-integration-test.postman_collection.json:1]: addressed all findings

**Review by ReneSchroederLJ — APPROVED** [review]
LGTM. Thank you for your contribution

**Review by mhellmeier — APPROVED** [review]
The >1000 LOC are fine for me since it is only an export of a Postman collection.

**Pull request merged**